server/src/Game/Collision.js

```diff
@@ -29,7 +29,7 @@ export default class Collision {
 			(this.window.h * 0.7 - this.window.h * 0.55) +
 			this.window.h * 0.55;
 
-		if (player.position.y + 60 >= noise && player.velocity.y > 0.3) {
+		if (player.position.y + 60 >= noise && player.velocity.y > 0.3 && player.velocity.x > 0.3) {
```

Comment:
It must be `OR` not `AND` as we need to do damage if players x velocity `OR` y velocity is greater than `0.3`
```suggestion
		if (player.position.y + 60 >= noise && (player.velocity.y > 0.3 || player.velocity.x > 0.3)) {
```